src/sep6/DepositProvider.ts

```diff
@@ -0,0 +1,65 @@
+import queryString from "query-string";
```

Comment:
I think we should not call this "sep6", it's too implementation-specific. I would call it Transfers or DepositWithdraw or something. The former is probably cleanest.

Reply:
Ah whoops! Renamed it everywhere but the folder.